client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/TabTables.tsx

```diff
@@ -60,10 +60,12 @@ export const QuantityTableComponent = ({
   const itemVariantsEnabled = useIsItemVariantsEnabled();
   const displayInDoses =
     !!preferences?.displayVaccineInDoses && !!item?.isVaccine;
-  const unitName = getPlural(
-    Formatter.sentenceCase(item?.unitName ? item.unitName : t('label.unit')),
-    2
+
+  const unitName = Formatter.sentenceCase(
+    item?.unitName ? item.unitName : t('label.unit')
```

Comment:
side quest - separate unit name from pluralise, as it is also used in singular form... before it said "Doses per Vials" rather than "Doses per vial"